gensrc/thrift/CloudConfiguration.thrift

```diff
@@ -19,7 +19,8 @@ enum TCloudType {
     DEFAULT,
     AWS,
     AZURE,
-    GCP
+    GCP,
+    OSS
```

Comment:
I think you need to use AliCloud rather than OSS, because others are cloud service vender names.

Reply:
+1

Reply:
done